### ISSUE TITLE
Configure your node as source of the blockchain instead of local storage

### ISSUE THREAD

**Issue by ronaldvdmeer** [input-output-hk/daedalus#2441]
I was wondering if its possible to simple look at my relay node instead of holding a local copy. If thats not currently possible this might be something for the feature request list.

**Comment by nikolaglumac** [comment]
Daedalus always runs it's own cardano-node instance which it uses to access of the blockchain.
Relay node is something else... Can you please further clarify your use-case? Why do you think it would be better/safer to use a relay?

**Comment by nikolaglumac** [comment]
@ronaldvdmeer I would really appreciate your input here 🙏

**Comment by ronaldvdmeer** [comment]
> @ronaldvdmeer I would really appreciate your input here 🙏

Well I have a fast node/relay in my homenetwork  already so startup times will improve because it doesn’t have to sync with a slow server. Sometimes it takes a long time (several minutes) to startup the client and I’m thinking it has something to do with the local node selecting a slow peer to sync with. 

Also it would be nice if it doesn’t have to sync anything. It just works with the information provided by the node in my network. That way no local storage is required beside the app itself. When the blockchain grows this might become an advantage. 

Hope this helps

**Comment by nikolaglumac** [comment]
Thanks for the clarification @ronaldvdmeer!
Would you agree with me that the feature request you are longing for is an option to configure the source of blockchain data?
E.g. it could also be a centralized service provider or your own remote instance?

**Comment by ronaldvdmeer** [comment]
I think there are two features.

1. being able to configure which node it will sync with
2. being able to not sync at all and work completely from another node.

**Comment by nikolaglumac** [comment]
Got it! Thanks!

**Comment by nikolaglumac** [comment]
I believe this issue is connected/related to https://github.com/input-output-hk/daedalus/issues/2413

**Comment by ronaldvdmeer** [comment]
Partially because it still needs to sync the whole blockchain. I tried this with my Mac by the way but somehow it didn’t work. Not sure why the topology file couldn’t be changed or found. Not sure anymore what the issue was

**Comment by nikolaglumac** [comment]
There is no easy nor nice way to change the topology at the moment...

**Comment by ronaldvdmeer** [comment]
Check! So one part of this feature request indeed is the same as the other. The other part of this feature request “being able to work of a relay without a local copy” is yet another feature

**Comment by nikolaglumac** [comment]
Agreed! This is why I have noted that the two are connected/related and not duplicates :-)

**Comment by ronaldvdmeer** [comment]
Perhaps I should change the title of this issue so it more reflects the difference

**Comment by nikolaglumac** [comment]
Yes please 🙏

**Comment by cedeon** [comment]
Is there any timeframe for this?  The blockchain is getting pretty big to run full nodes locally and I have a perfectly and permanently sync'd cardano-wallet docker container running on my server exposing port 8090 over http.  Would it be hard to allow daedalus to switch cardano-wallet/ cardano-node back ends instead of running these locally?  I'd really <3 this feature!
